### PR TITLE
Fix #59 and #56 : reduce the weight of the Docker image + `shm` note added

### LINKED ISSUE #59: Reduce image weigth by removing `/var/lib/apt/lists/*`
The directory `/var/lib/apt/lists/*` is about 40 Mo but theses files are cached from the `apt-get update`. It could be remove.

See [examples by Jessica Frazzel](https://github.com/jfrazelle/dockerfiles/search?utf8=%E2%9C%93&q=%2Fvar%2Flib%2Fapt%2Flists%2F).

### PR DESCRIPTION
**Pull request by auzias**
* Removing this directory (and its sub-ones) reduce the image weight.
* A note about `shm` has been added in the `README.md`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,4 +21,5 @@ docker run -d -v /tmp/.X11-unix/:/tmp/.X11-unix/ \
               -e DISPLAY=${DISPLAY} \
               jamesnetherton/docker-atom-editor
 ```
+Note that `-v /dev/shm:/dev/shm` may be optionnal and can be replaced by `--shm-size="<number><unit>"`.
```
Comment (jamesnetherton) [inline README.md]: Typo: 'optionnal'

Comment (auzias) [inline README.md]: Thanks! Cleared :+1:

File: Dockerfile [hunk 2]
```diff
@@ -20,8 +21,8 @@ RUN apt-get update && \
                     libnss3 \
                     gvfs-bin \
                     xdg-utils \
-                    python -y --no-install-recommends && \
-    apt-get clean && \
+                    python && \
+    apt-get clean && rm -rf /var/lib/apt/lists/* \
```
Comment (jamesnetherton) [inline Dockerfile]: You're missing a '&&' after the rm of the apt lists.

Comment (auzias) [inline Dockerfile]: Damn! I should I have at least build it. I forgot you put it in a sole layer. Corrected.

**Comment by jamesnetherton** [comment]
Thanks (again)!

**Pull request merged**